### PR TITLE
Update title of Basic BitModal section of Modal demo (#5673)

### LINKED ISSUE #5673: Update the title of the `Basic BitModal` section of the `Modal` demo page
The current title of this section is too long and not appropriate:

image

This title should be changed to `Basic`.

The demo file name is `BitModalDemo.razor` in the `Bit.BlazorUI.Demo.Client.Core` project.

### PR DESCRIPTION
**Pull request by reza-dehghani**
This closes #5673.

### CODE REVIEW

File: src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor [hunk 1]
```diff
@@ -278,4 +278,4 @@
         IsOpenInPosition = true;
         position = positionValue;
     }
-}
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor:281]: revert this change plz, do not remove this empty new line at the end of the file.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor (hunk 1 above)
Comment (reza-dehghani) [reply src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor:281]: Hello, ok.

Comment (msynk) [reply src/BlazorUI/Demo/Client/Core/Pages/Components/Modal/BitModalDemo.razor:281]: you didn't change anything!

**Comment by msynk** [comment]
Thanks for the update, but you didn't change anything. could you plz double-check and push a new commit?
